src/test/java/com/rackspace/ceres/app/services/MetricDeletionServiceTest.java

```diff
@@ -122,11 +122,59 @@ public void testDeleteMetricsByTenantId() {
     metricDeletionService.deleteMetrics(tenantId, "", null,
         Instant.now().minusSeconds(60), Instant.now()).then().block();
 
-    assertViaQuery(tenantId, timeSlotPartitioner.rawTimeSlot(currentTime), seriesSetHash,
-        metricName);
+    //validate data raw
+    assertQueryRawViaQuery(tenantId, timeSlotPartitioner.rawTimeSlot(currentTime), 0);
+
+    //validate series_set_hashes
+    assertSeriesSetHashesViaQuery(tenantId, seriesSetHash, 1);
+
+    //validate series_sets
+    assertSeriesSetViaQuery(tenantId, metricName, 4);
+
+    //validate metric_names
+    assertMetricNamesViaQuery(tenantId, metricName, 1);
+  }
+
+  @Test
+  public void testDeleteMetricsByTenantIdAndWithoutStartTime() {
```

Comment:
Here I would like to see that all the data is missing after a delete.

Reply:
yes, that's what is being checked as of now: https://github.com/racker/ceres/blob/42d90969cbb4a8d45384acbe46d1c7c850e0911b/src/test/java/com/rackspace/ceres/app/services/MetricDeletionServiceTest.java#L168

here, for all the tables in Cassandra expected rowCount is passed as 0 just to check all metric data along with its metadata is deleted